src/python/sortingview/gui/extensions/mountainview/MVSortingView/CurationControl.tsx

```diff
@@ -130,23 +142,35 @@ const CurationControl: FunctionComponent<Props & SizeMeProps> = ({ selection, se
             <Paper style={paperStyle} key="merge">                
                 Merge:
                 {
-                    selectedUnitIds.length >= 2 && <button key="merge" onClick={handleMergeSelected}>Merge selected units: {selectedUnitIds.join(', ')}</button>
+                    (selectedUnitIds.length >= 2 && !unitsAreInMergeGroups(selectedUnitIds)) &&
+                        <button key="merge" onClick={handleMergeSelected} disabled={curation?.isClosed || false}>
+                            Merge selected units: {selectedUnitIds.join(', ')}
+                        </button>
                 }
                 {
-                    (selectedUnitIds.length > 0 && unitsAreInMergeGroups(selectedUnitIds)) && <button key="unmerge" onClick={handleUnmergeSelected}>Unmerge units: {selectedUnitIds.join(', ')}</button>
+                    (selectedUnitIds.length > 0 && unitsAreInMergeGroups(selectedUnitIds)) &&
+                        <button key="unmerge" onClick={handleUnmergeSelected} disabled={curation?.isClosed || false}>
+                            Unmerge units: {selectedUnitIds.join(', ')}
+                        </button>
                 }
                 <span style={{whiteSpace: 'nowrap'}}>
                     <Checkbox checked={selection.applyMerges || false} onClick={handleToggleApplyMerges}/> Apply merges
                 </span>
             </Paper>
+            <Button
+                color={ curation?.isClosed ? "primary" : "secondary" }
+                variant={"contained"}
+                onClick={() => {handleToggleCurationClosed(curation?.isClosed || false )}}>
```

Comment:
Let's avoid having an inline function here. Here's how to avoid it:

* use instead `onClick={handleToggleCurationClosed}`
* Remove the argument of handleToggleCurationClosed
* Instead have curationIsClosed be a dependency of the handleToggleCurationClosed callback. It can be computed before the callback hook via: `const curationIsClosed = curation ? curation.isClosed : false`